modules/platform/nuxeo-automation/nuxeo-automation-scripting/src/main/java/org/nuxeo/automation/scripting/internals/AutomationScriptingServiceImpl.java

```diff
@@ -31,24 +32,19 @@
 import java.lang.reflect.Method;
 import java.lang.reflect.Proxy;
 import java.util.HashSet;
+import java.util.Map;
 import java.util.Set;
+import java.util.function.Predicate;
+
+import javax.script.*;
```

Comment:
i suspect wildcard imports are banned by static analysis / code style, right? should i apply a fix for this?